config/application.rb

```diff
@@ -191,16 +191,16 @@ class Application < Rails::Application
 
     # Check that the loggers setting exist to configure the app and sql loggers
     Foreman::Logging.add_loggers((SETTINGS[:loggers] || {}).reverse_merge(
-      :app => {:enabled => true},
-      :audit => {:enabled => true},
-      :ldap => {:enabled => false},
-      :permissions => {:enabled => false},
-      :proxy => {:enabled => false},
-      :sql => {:enabled => false},
-      :templates => {:enabled => true},
-      :notifications => {:enabled => true},
-      :background => {:enabled => true},
-      :dynflow => {:enabled => true}
+                                   :app => {:enabled => true},
+                                   :audit => {:enabled => true},
+                                   :ldap => {:enabled => false},
+                                   :permissions => {:enabled => false},
+                                   :proxy => {:enabled => false},
+                                   :sql => {:enabled => false},
+                                   :templates => {:enabled => true},
+                                   :notifications => {:enabled => true},
+                                   :background => {:enabled => true},
+                                   :dynflow => {:enabled => true}
```

Comment:
Typical example of bad practice - change of the first line (193) leads to full reformatting, making more conflicts and overall this does not improve readability at all - IMHO makes that harder because this will likely hit 80th column more often.

Reply:
I'd argue the actual issue here is that the defaults hash is all passed directly to the reverse merge instead of being defined as a well-named variable first.